blob/gcsblob/gcsblob_test.go

```diff
@@ -180,7 +180,7 @@ func (verifyContentLanguage) AttributesCheck(attrs *blob.Attributes) error {
 }
 
 func (verifyContentLanguage) ReaderCheck(r *blob.Reader) error {
-	var rr storage.Reader
```

Comment:
Actually, you also need to update this test:
https://github.com/google/go-cloud/blob/master/blob/example_test.go#L451

Reply:
Done. I would not have found that.